### DISCUSSION TITLE
Question: Does the checkpoint matter at all when running this script?

### DISCUSSION THREAD

**Discussion by mr-september** [Coyote-A/ultimate-upscale-for-automatic1111#48] [Q&A] [answered]
As title, does the "Stable Diffusion checkpoint" matter at all when running this script?

Or the prompts?

**Comment by Jopezzia** [comment, marked as answer]
Yes, the ckpt matters. It's responsible for the style, as it works through inpaint. Prompt is also responsible for the morph
